anaSbitReadout.py

```diff
@@ -0,0 +1,376 @@
+#!/bin/env python
+"""
+    Script to analyze sbitreadout data
+    By: Caterina Aruta (caterina.aruta@cern.ch) and Francesco Ivone (francesco.ivone@cern.ch)
+    """
```

Comment:
This needs to be redone such that it is compatible `sphinx` so that automatic documentation is made.

See examples:

- [gemPlotter.py](https://github.com/cms-gem-daq-project/gem-plotting-tools/blob/develop/macros/gemPlotter.py),
- [gemPlotter.rst](https://github.com/cms-gem-daq-project/gem-plotting-tools/blob/develop/doc/man/gemPlotter.rst), and
- [man-index.rst](https://github.com/cms-gem-daq-project/gem-plotting-tools/blob/develop/doc/man-index.rst)

Reply:
I've adapted the gemPlotter.py to make an auto documentation. However I've left the first 4 lines as they were (my forgetfulness). Let me know if it needs to be changed